fleetctl/fleetctl.go

```diff
@@ -669,6 +669,81 @@ func createUnit(name string, uf *unit.UnitFile) (*schema.Unit, error) {
 	return &u, nil
 }
 
+// checkReplaceUnitState checks if the unit should be replaced
+// It takes a Unit object as a parameter
+// It returns 0 on success and if the unit should be replaced, 1 if the
+// unit should not be replaced; and any error acountered
+func checkReplaceUnitState(unit *schema.Unit) (int, error) {
```

Comment:
Sorry, this was not the best place to put this remark: I actually totally agree that the function name should not be capitalised. (And by Go conventions, the sentence actually has to start with the function name... So it basically has to be ugly.)

My remark about sentences starting with a capital letter was actually referring to some of the other comments, where some sentences not starting with an identifier were not capitalised either...